internal/pkg/docker/docker.go

```diff
@@ -6,33 +6,35 @@ package docker
 
 import (
 	"fmt"
+	"path/filepath"
 	"strings"
 
 	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/term/command"
 	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/term/log"
 )
 
-// Service wraps a runner.
-type Service struct {
-	runner runner
+// Runner wraps a runner.
```

Comment:
nit: since `runner` is a private interface what do you think if we write:
```suggestion
// Runner represents a command that can be run.
```